### ISSUE TITLE
Right Axis Line using same path effect as x axis

### ISSUE THREAD

**Issue by DiegoMacBD** [nativescript-community/ui-chart#44]
**Summary**
Right now I'm trying to draw two different things in my chart: 

1.- A grid on the x Axis that looks like this:

![image](https://user-images.githubusercontent.com/84344373/143334287-cc24315f-8156-45d2-aeaa-6d185eb2471e.png)

using: 
`   
xAxis.enableGridDashedLine(10, height, 0);
`

2.- A Line all the way to the right of the chart:

![image](https://user-images.githubusercontent.com/84344373/143334419-fc1b3a31-8c46-4b46-9fb6-4e7235b687ef.png)
using 
```
    rightAxis.setEnabled(true);
    rightAxis.setAxisMaximum(this.graphMaximum);
    rightAxis.setAxisMinimum(this.graphMinimum);
    rightAxis.setDrawAxisLine(true);
    rightAxis.setAxisLineWidth(2);
    rightAxis.setAxisLineColor(this.colors.label);
```

The problem is that if I set a path effect for the x Axis (the little lines at the bottom of the chart), the right axis line also gets that path effect, ending up looking like this:

![image](https://user-images.githubusercontent.com/84344373/143334638-f4a97427-eabd-4a25-90b0-975b3c656bd4.png)

Is this intended or a bug in the code? if it is intended, is there a workaround for it? Thank you.

**Comment by farfromrefug** [comment]
@DiegoMacBD i cant see to see anything looking at the code. Could you create a repro sample? (it is abug)

**Comment by farfromrefug** [comment]
@DiegoMacBD should be fixed in 1.2.13

**Comment by DiegoMacBD** [comment]
Thank you so much, for all the help, indeed it is fixed :)
